Setup/Patch/Data/AddProductAttributes.php

```diff
@@ -0,0 +1,175 @@
+                'is_filterable_in_grid' => false,
+                'option' => ''
+            ]
+        );
+
+        $eavSetup->addAttribute(
+            Product::ENTITY,
+            ProductHelper::MAGENTO_PRODUCT_ATTRIBUTE_VOTE_COUNT,
+            [
+                'type' => 'int',
+                'label' => 'Lipscore Votes Count',
+                'input' => 'text',
+                'source' => '',
+                'frontend' => '',
+                'required' => false,
+                'backend' => '',
+                'sort_order' => '150',
+                'global' => ScopedAttributeInterface::SCOPE_GLOBAL,
+                'default' => null,
+                'visible' => true,
+                'user_defined' => true,
+                'searchable' => true,
+                'filterable' => true,
+                'comparable' => true,
+                'visible_on_front' => true,
+                'unique' => false,
+                'group' => 'Lipscore Ratings & Reviews',
+                'used_in_product_listing' => true,
+                'is_used_in_grid' => true,
+                'is_visible_in_grid' => false,
+                'is_filterable_in_grid' => false,
+                'option' => ''
+            ]
+        );
+
+        $eavSetup->addAttribute(
+            Product::ENTITY,
+            ProductHelper::MAGENTO_PRODUCT_ATTRIBUTE_REVIEW_COUNT,
+            [
+                'type' => 'int',
+                'label' => 'Lipscore Reviews Count',
+                'input' => 'text',
+                'source' => '',
+                'frontend' => '',
+                'required' => false,
+                'backend' => '',
+                'sort_order' => '200',
+                'global' => ScopedAttributeInterface::SCOPE_GLOBAL,
+                'default' => null,
+                'visible' => true,
+                'user_defined' => true,
+                'searchable' => true,
+                'filterable' => true,
+                'comparable' => true,
+                'visible_on_front' => true,
+                'unique' => false,
+                'group' => 'Lipscore Ratings & Reviews',
+                'used_in_product_listing' => true,
+                'is_used_in_grid' => true,
+                'is_visible_in_grid' => false,
+                'is_filterable_in_grid' => false,
+                'option' => ''
+            ]
+        );
+
+        $this->moduleDataSetup->getConnection()->endSetup();
+    }
+
+    protected function createAttributeGroup($attributeGroupName)
+    {
+        $attributeSetId = $this->product->getDefaultAttributeSetId();
+
+        $attributeGroup = $this->attributeGroupFactory->create();
+        $attributeGroup->setAttributeSetId($attributeSetId);
+        $attributeGroup->setAttributeGroupName($attributeGroupName);
+
+        $this->attributeGroupRepository->save($attributeGroup);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getAliases()
```

Comment:
we usually use the next approach on listing methods in classes:
- all the public methods first (an interface of a class)
- all the protected methods after public ones (internal functionality)